Process.c

```diff
@@ -659,6 +660,44 @@ void Process_writeCommand(const Process* this, int attr, int baseAttr, RichStrin
 
       RichString_setAttrn(str, hl->attr, strStart + hl->offset, hl->length);
    }
+
+   if (this->settings->shadowDistPathPrefix && mergedCommand[0] == '/') {
+      #define CHECK_AND_MARK(prefix_)                                                         \
+         if (String_startsWith(mergedCommand, prefix_)) {                                     \
+            RichString_setAttrn(str, CRT_colors[PROCESS_SHADOW], strStart, strlen(prefix_));  \
+            break;                                                                            \
+         } else (void)0
+
+      switch (mergedCommand[1]) {
+         case 'b':
+            CHECK_AND_MARK("/bin/");
+            break;
+         case 'l':
+            CHECK_AND_MARK("/lib/");
```

Comment:
CHECK_AND_MARK("/lib64/");

... is common on Fedora and probably other distros too.

Reply:
Also includes `/lib32/` and `/libx32/`,